src/wallet/coincontrol.h

```diff
@@ -41,9 +43,9 @@ class CCoinControl
         fAllowOtherInputs = false;
         fAllowWatchOnly = false;
         setSelected.clear();
-        nFeeRate = CFeeRate(0);
+        m_feerate = boost::none;
```

Comment:
In commit "Refactor to use CoinControl in GetMinimumFee and FeeBumper"

Maybe replace ` = boost::none` with `.reset()` here and other places to remove boost reference and make it a little easier to port to `std::optional` in the future.